docs/Developers/Debug_Bears.rst

```diff
@@ -0,0 +1,52 @@
+Debug Bears
+===========
+
+This document provide a brief overview of coala's Debugging Interface.
+Debugging Interface will help user to debug the Bear code and move through
+it using ``pdb`` Interface
+
+There are two way to invoke the coala's Debgger
+
+Command Line Interface
+----------------------
+
+User can specify the bear they want to debug using ``--debug-bears`` i.e.,
+
+.. code:: shell
+
+    $ coala -b PEP8Bear,MypyBear -f <filename> --debug-bears PEP8Bear
+
+If bear name is not specified in ``--debug-bears`` argument then it will by
+default debug all the bears passed through ``--bears`` or ``-b`` argument
+**i.e.,**
+
+.. code:: shell
+
+    $ coala -b PEP8Bear,MypyBear -f <filename> --debug-bears
```

Comment:
Why `i.e.` before? I thought you explain something now, but this is just another command^^